src/Symfony/Component/PasswordHasher/Hasher/PasswordHasherFactoryInterface.php

```diff
@@ -26,9 +25,7 @@ interface PasswordHasherFactoryInterface
     /**
      * Returns the password hasher to use for the given user.
      *
-     * @param PasswordAuthenticatedUserInterface|UserInterface|string $user A PasswordAuthenticatedUserInterface/UserInterface instance or a class name
```

Comment:
the description was still explaining what type of string was supported

Reply:
any string is supported actually (named hashers)